README.md

```diff
@@ -30,16 +30,6 @@ We recommend you start from there.
 
 Our documentation assumes that you have working knowledge of the Modelica language, are familiar with a Modelica-compliant modeling and simulation environment, that you have proeficient knowledge on power system steady state analysis (i.e., the so called "power flow"), and knowledge on power system dynamic modeling (i.e., the called "transient" and "small-signal" stability). As such, the documentation is limited, and aims to provide very consice information for people who fulfill the requirements above. If you do not fulfill these requirements, we recommend that you first get acquainted with both Modelica and Power Systems.
 
-
-## Contributing:
-Contributions to the library are welcome and can be submitted in the form of pull requests to this repository.
-
-Please consult the [contribution guidelines](http://openipsl.readthedocs.io/en/latest/community/contributing.html).
-
-More information about our contributors can be found over [here](http://openipsl.readthedocs.io/en/latest/community/community-home.html).
-
-Don't hesitate to contact us.
-
 ## Citing OpenIPSL in Publications:
 If you use OpenIPSL in your work or research, all we ask you in exchange is that you **cite the reference publications**, according to your use.
 Please consult our [Publication List](http://openipsl.readthedocs.io/en/latest/publications.html), for browsing the reference publications.
```

Comment:
Do NOT link to the readthedocs page. That one should actually be removed and never mentioned again since we do not keep that up to date.